### PR TITLE
Add Pre-LayerNorm and fix MolHIV script

### PR DESCRIPTION
**Pull request by shiyu1994**
This PR is to close the #90. We add pre-layernorm option and fix the script for MolHIV fine tuning to get the correct results.

### CODE REVIEW

File: examples/property_prediction/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ GCN-VN          | 4.9M    | 0.1225    | 0.1485    |
 GIN-VN          | 6.7M    | 0.1150    | 0.1395    |
 Graphormer-Small| 12.5M   | 0.0778    | 0.1264    |
 Graphormer   | 47.1M   | 0.0582    | 0.1234    |
-Graphormer-v2   | 47.1M   | 0.0309    | **0.1201**    |
+Graphormer-v2   | 48.3M   | 0.0309    | **0.1201**    |
```
Comment (zhengsx) [inline examples/property_prediction/README.md:35]: May report consistent number with our paper?

File: examples/property_prediction/README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ PHC-GNN          | 111K    | 79.34      |
 DeeperGCN-FLAG          | 532K    | 79.42      |
 DGN          | 114K    | 79.70      |
 Graphormer   | 47.0M   | 80.51      |
-Graphormer-v2   | 47.1M   | **81.28**      |
+Graphormer-v2   | 48.3M   | **80.56**      |
```
Comment (zhengsx) [inline examples/property_prediction/README.md:58]: The difference is marginal, report the latest is good enough i think

File: examples/property_prediction/README.md (hunk 1 above)
Comment (shiyu1994) [reply examples/property_prediction/README.md:35]: In paper, the number of parameters is 48.3M for base models. I think 47.1M is for Graphormer V1.

File: examples/property_prediction/README.md (hunk 2 above)
Comment (zhengsx) [reply examples/property_prediction/README.md:58]: how about remove the -v2 suffix here

File: examples/property_prediction/README.md (hunk 1 above)
Comment (zhengsx) [reply examples/property_prediction/README.md:35]: I mean the MAE result

Comment (shiyu1994) [reply examples/property_prediction/README.md:35]: Ok. I'll fix that.

File: examples/property_prediction/hiv_pre.sh [hunk 3]
```diff
@@ -34,8 +32,9 @@ fairseq-train \
 --encoder-attention-heads 32 \
 --max-epoch $max_epoch \
 --save-dir ./ckpts \
---pretrained-model-name pcqm4mv1_graphormer_base \
+--pretrained-model-name pcqm4mv1_graphormer_base_for_molhiv \
```
[the remark this answers is not in the document]
Comment (zhengsx) [reply examples/property_prediction/hiv_pre.sh:35]: the previous one is for reproduction of the PCQ experiment and the later is for HIV.

**Pull request merged**